.vscode/tasks.json

```diff
@@ -0,0 +1,38 @@
+// https://code.visualstudio.com/docs/editor/variables-reference
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Rails - server",
+      "type": "shell",
+      "command": "RAILS_ENV=development $(which bundle) exec rails server -b 0.0.0.0 -p 3000"
+    },
+    {
+      "label": "RSpec - all",
+      "type": "shell",
+      "command": "RAILS_ENV=test $(which bundle) exec rspec"
+    },
+    {
+      "label": "RSpec - unit",
+      "type": "shell",
+      "command": "RAILS_ENV=test $(which bundle) exec rspec --exclude-pattern spec/{features}/**/*_spec.rb"
```

Comment:
please wrap the pattern in "", otherwise shell globs will cause rspec to miss some files that should be tested